### PR TITLE
commiting perl-file-find-rule

### PR DESCRIPTION
**Pull request by jerowe**
* [ ] I have read the guidelines above.
* [x] This PR adds a new recipe.
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/perl-number-compare/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+requirements:
+  build:
+    - perl-threaded
+    - perl-extutils-makemaker
+
+  run:
+    - perl-threaded
+
+# test:
+  # Perl 'use' tests
```
Comment (bgruening) [inline recipes/perl-number-compare/meta.yaml]: Tests are missing for this one.

File: recipes/perl-text-glob/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    - perl-threaded
+    - perl-module-build
+  run:
+    - perl-threaded
+
+# test:
+  # Perl 'use' tests
+  # imports:
+
+  # You can also put a file called run_test.pl (or run_test.py) in the recipe
```
Comment (bgruening) [inline recipes/perl-text-glob/meta.yaml]: Tests are missing.

**Comment by bgruening** [comment]
Nice additions! Can you please add tests to the two packages. Thanks

**Comment by jerowe** [comment]
Added the tests.

**Pull request merged**